torch/functional.py

```diff
@@ -488,19 +501,22 @@ def stft(input, n_fft, hop_length=None, win_length=None, window=None,
         pad = int(n_fft // 2)
         input = F.pad(input.view(extended_shape), (pad, pad), pad_mode)
         input = input.view(input.shape[-signal_dim:])
-    return _VF.stft(input, n_fft, hop_length, win_length, window, normalized, onesided)  # type: ignore
```

Comment:
Same question here about why we have Python bindings instead of relying on autogenerated bindings. Also: how does that block of Python code work with the C++ API? Something about this is either confusing me or seems fishy.

Reply:
Yes, I agree this is problematic for the C++ api. It currently doesn't get support for `center` and `pad_mode`.

Reply:
We have this issue in other places, too, unfortunately.